packages/iframe-stamper/src/index.ts

```diff
@@ -132,9 +134,11 @@ export class IframeStamper {
 
     let iframe = window.document.createElement("iframe");
 
+    this.trustedParentOrigin = window.origin;
```

Comment:
A thought I'm having is that this won't necessarily work in react native (what would "window.origin" resolve to?). But I think that's fine because RN apps don't use iframe stampers do they?

Another thing worth checking is TG apps. But again, they probably don't use iframe stampers?

Reply:
I removed this entirely in favor of the MessageChannel implementation